app/src/main/res/animator/u1f60e_left_shade_animator.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:ordering="sequentially">
```

Comment:
I just realised that we don't need this to be a set, the root could be directly a `objectAnimator`. It's definitely my bad, I should have done that in the other existing animators.